### PR TITLE
Draft: WIP: work on section 3 of secret cheatsheet

### PR DESCRIPTION
**Pull request by commjoen**
Thank you for submitting a Pull Request (PR) to the Cheat Sheet Series. 

This PR updates issue #124 >.

### CODE REVIEW

File: cheatsheets_draft/Secrets_Management_CheatSheet.md [hunk 1]
```diff
@@ -154,16 +155,47 @@ Given that the CI/CD tooling heavily consume secrets, it is key that the pipelin
 - Have proper authentication, authorization and accounting in place.
 - Make sure that pipelines can only be created by means of an approved process, including MR/PR steps to make sure that a created pipeline is security-reviewed.
 
-### 3.2. Where should a secret be?
+
+### 3.2 Where should a secret be?
+
+- DO YOU PUT THE SECRET IN GIT PROVISIONGIN SYSTEM?J
```
Comment (darkgl0w) [inline cheatsheets_draft/Secrets_Management_CheatSheet.md:161]:
```suggestion
- DO YOU PUT THE SECRET IN GIT PROVISIONING SYSTEM?
```

File: cheatsheets_draft/Secrets_Management_CheatSheet.md [hunk 2]
```diff
@@ -154,16 +155,47 @@ Given that the CI/CD tooling heavily consume secrets, it is key that the pipelin
...
+
+EXPLAIN DO'S AND DON'TS
+
+#### 3.2.3 Storing it in a secrets management system
+
+EXPLAIN DO'S AND DON'TS
+
+### 3.3: Authentication and Authorization
+
+HOW DOES A PIPELIEN AUTHENTICATE? HOW DO YOU KNOW AUTHORIZAITON IS OK?
```
Comment (darkgl0w) [inline cheatsheets_draft/Secrets_Management_CheatSheet.md:182]:
```suggestion
HOW DOES A PIPELINE AUTHENTICATE? HOW DO YOU KNOW AUTHORIZATION IS OK?
```

File: cheatsheets_draft/Secrets_Management_CheatSheet.md [hunk 3]
```diff
@@ -154,16 +155,47 @@ Given that the CI/CD tooling heavily consume secrets, it is key that the pipelin
...
 
-### 3.4. Deployment
+### 3.4: Logging and accounting
+ 
+HOW CAN YOU TELL WHO ACCESSED THE SECRET WITH THE PIPELINE?
```
Comment (darkgl0w) [inline cheatsheets_draft/Secrets_Management_CheatSheet.md:186]:
```suggestion
HOW CAN YOU TELL WHO ACCESSED THE SECRET WITHIN THE PIPELINE?
```

**Review by darkgl0w — COMMENTED** [review]
Hello.
There are a few minor typos.

**Review by jmanico — APPROVED** [review]
so far so good

File: cheatsheets_draft/Secrets_Management_CheatSheet.md (hunk 2 above)
Comment (commjoen) [reply cheatsheets_draft/Secrets_Management_CheatSheet.md:180]: removed the text as this was a placeholder for the actual text.

**Pull request merged**